src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    public String draw();

```

Comment:
```suggestion
    public void draw();
```
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/b0f5f090-4e3a-4392-8582-f61207192d28)